srcpkgs/popt/template

```diff
@@ -1,16 +1,20 @@
 # Template file for 'popt'
 pkgname=popt
-version=1.16
-revision=7
+version=1.18
+revision=1
 build_style=gnu-configure
+hostmakedepends="autoconf tar automake libtool gettext-devel"
 short_desc="Command line option parsing library"
 maintainer="Orphaned <orphan@voidlinux.org>"
-homepage="http://rpm5.org"
 license="MIT"
-#distfiles="http://rpm5.org/files/$pkgname/$pkgname-$version.tar.gz"
-distfiles="https://fossies.org/linux/misc/popt-1.16.tar.gz"
-checksum=e728ed296fe9f069a0e005003c3d6b2dde3d9cad453422a10d6558616d304cc8
+homepage="https://rpm.org/"
```

Comment:
This doesn't really seem like the right home page because the package is only mentioned in passing in the "NEWS" section. I'd advocate either https://github.com/rpm-software-management/popt or http://ftp.rpm.org/popt/ which the Github repo links as the home page.

Reply:
I disagree :)
popt development was done under the umbrella of rpm5.org and now it is under the umbrella of the rpm developers at rpm.org. The previous homepage was rpm5.org, the current homepage should be rpm.org. Your first proposal is  actually a repo page in the rpm gh organisation, which is linked from **rpm.org**, the second is a directory in the **rpm.org** ftp server. I may be confused (it is known to happen :P) but I usually try to find a proper http site for the project and if that is missing, I default to the repo. Having a more general web site as home page is not unprecedented (see all packages belonging to certan DEs).

Reply:
That's a reasonable view.